tests/client/amo/components/TestAddonReview.js

```diff
@@ -92,8 +94,8 @@ describe('AddonReview', () => {
           apiState: signedInApiState,
         });
 
-        // Make sure the overlay was hidden after finishing.
-        assert.ok(overlayCard.hide.called);
+        assert.ok(onReviewSubmitted.called,
+                  'onReviewSubmitted() should have been called after updating');
```

Comment:
Sure. That would be easy to add a lint rule for so we should do it. It's not easy to add lint rules for vertical alignment.